components/p3a/message_manager_unittest.cc

```diff
@@ -58,8 +58,40 @@ class P3AMessageManagerTest : public testing::Test,
                 &url_loader_factory_)) {}
 
   std::optional<MetricLogType> GetDynamicMetricLogType(
-      const std::string& histogram_name) const override {
-    return std::optional<MetricLogType>();
+      std::string_view histogram_name) const override {
+    return std::nullopt;
+  }
+
+  const MetricConfig* GetMetricConfig(
+      std::string_view histogram_name) const override {
+    const std::optional<MetricConfig>* metric_config = nullptr;
+
+    auto it = p3a::kCollectedTypicalHistograms.find(histogram_name);
```

Comment:
is it copypaste? can't we make a helper function and invoke it from two call sites?

Reply:
good point